src/platform/user/profile/vap-svc/tests/addressValidationMessages.unit.spec.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
 import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 import sinon from 'sinon';
 import {
   ADDRESS_VALIDATION_MESSAGES,
   ADDRESS_VALIDATION_TYPES,
-} from '../../profile/constants/addressValidationMessages';
+} from '../constants/addressValidationMessages';
 
 describe('ADDRESS_VALIDATION_MESSAGES object', () => {
-  it('has a valid data object for each validation type', () => {
+  it('has a valid data object for each ADDRESS_VALIDATION_TYPE', () => {
     const validationTypes = Object.values(ADDRESS_VALIDATION_TYPES);
     validationTypes.forEach(type => {
       const messageData = ADDRESS_VALIDATION_MESSAGES[type];
       const editSpy = sinon.spy();
       expect(typeof messageData).to.equal('object');
       expect(typeof messageData.headline).to.equal('string');
       expect(typeof messageData.ModalText).to.equal('function');
-      const modalTextComponent = shallow(
+      const modalTextComponent = mount(
         <messageData.ModalText editFunction={editSpy} />,
       );
-      modalTextComponent.find('a').simulate('click');
+      modalTextComponent.find('button').simulate('click');
```

Comment:
fun fact... these tests were never being run because the file was not named correctly. It was missing the `unti.spec.js` suffix. So I gave the file the proper suffix, moved it to the vap-svc dir, and also fixed the test since it was failing.